content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -426,7 +427,8 @@ transitions a lifecycle stage as follows. Feature gates must function for no les
 
 **Rule #9: Deprecated feature gates must respond with a warning when used. When a feature gate
 is deprecated it must be documented in both in the release notes and the corresponding CLI help.
-Both warnings and documentation must indicate whether a feature gate is non-operational.**
+Both warnings and documentation must indicate whether a feature gate is non-operational.
+Feature gate must be disabled by default for the deprecation window.**
```

Comment:
```suggestion
Throughout the deprecation period, the feature gate must be disabled by default.
```
BTW is this true? If we had a legacy-mode feature gate such as `LegacyNodeRoleBehavior` and then wanted to deprecate the feature gate, how would that work?

Reply:
yeah, feature gates *can* be false by default during the deprecation window, but I don't think it's **required**

Reply:
if feature gate is still in code, it means deprecation happens from alpha or beta. If it is alpha, then nothing changes, feature gate keep being disabled. If it was in beta, what would be the reason to keep feature gate enabled by default? Since we want to discourage use of the feature, we want to make sure to not have it enabled by default on upgrade. So it will be an intentional step to re-enable it. I can change must to should if this is better.

As for `LegacyNodeRoleBehavior`, it is already GA. Deprecation is not possible after the GA as per text above in this section. So there will be a separate deprecation feature gate.

Reply:
I wanted to suggest a thought experiment: imagine there is a brand new feature gate `LegacyPersistentVolumeBehavior', and like `LegacyNodeRoleBehavior` it starts life as alpha, but because of circumstances, we set `LegacyPersistentVolumeBehavior' true by default.

Then, as a project, we have a change of heart. We decide not to fix the imaginary defect in the PV API, and instead we'll live with its shortcomings. We deprecate `LegacyPersistentVolumeBehavior`, leaving it true. Until that deprecation is complete you can set it false and get the new, alpha behavior for PVs, but you're on marked time. We never default `LegacyPersistentVolumeBehavior` to false but we do eventually take that feature gate out of the CLI options for the binaries we ship and the code they build from.

When I think about this experiment, it makes me not want to say:
> Throughout the deprecation period, the feature gate must be disabled by default.

Reply:
as per previous section:

>  * Alpha: the feature gate is disabled by default and can be enabled by the user.
>  * Beta: the feature gate is enabled by default and can be disabled by the user.
>  * GA: the feature gate is deprecated (see ["Deprecation"](#deprecation)) and becomes
>  non-operational.
>  * GA, deprecation window complete: the feature gate is removed and calls to it are
>  no longer accepted.

I used disabled/enabled with the same meaning, not with the meaning of actual value being "true" or "false".

Reply:
the natural way to read disabled/enabled is to map to false/true... I still don't think requiring a deprecated gate to be disabled/false is universally necessary